packages/quantic/force-app/main/default/lwc/quanticFacet/quanticFacet.js

```diff
@@ -76,10 +76,10 @@ export default class QuanticFacet extends LightningElement {
    *   - `occurrences`
    *   - `automatic`
    * @api
-   * @type  {'score' | 'alphanumeric' | 'occurrences' | 'automatic'}
-   * @defaultValue `'automatic'`
+   * @type  {'score' | 'alphanumeric' | 'occurrences'}
+   * @defaultValue `'alphanumeric'`
```

Comment:
We should use 'score' as a default instead of alphanumeric though.

Reply:
Changed @api sortCriteria to score for FacetValue